### ISSUE TITLE
Please add code fix: Remove unnecessary initialization

### ISSUE THREAD

**Issue by wmjordan** [code-cracker/code-cracker#718]
In the following code:

    int num = 0;
    num = comparer.Compare (this.m_Item1, tuple.m_Item1);

`num = 0` is redundant since it immediately get assigned by subsequent statement. The code can be simplified as the following.

    int num = comparer.Compare (this.m_Item1, tuple.m_Item1);

Please add a code fix for this case.

**Comment by kidchenko** [comment]
+1

**Comment by giggio** [comment]
Makes sense to me. But only to assignments to literals or existing variables/fields/properties, as constructions and methods could have side effects.
